packages/support/src/Services/RelationshipJoiner.php

```diff
@@ -91,6 +91,12 @@ function (array $where) use ($relationship): bool {
             $relationshipQuery
                 ->distinct()
                 ->select($relationshipQuery->getModel()->getTable() . '.*');
+
+            foreach ($relationshipQuery->getQuery()->orders as $order) {
+                if (array_key_exists('column', $order)) {
+                    $relationshipQuery->addSelect($order['column']);
```

Comment:
Are the `$order['column']` values always qualified with the table name?

If so, can we add a new condition to ONLY `addSelect()` IF the column name does NOT start with `"{$relationshipQuery->getModel()->getTable()}".`

Because we already have a select for `->select($relationshipQuery->getModel()->getTable() . '.*')` which should catch some of these?

Reply:
Just checked the code for the eloquent builder and it seems that for an order query it always calls `$this->model->getQualifiedKeyName()` which adds the table name in front the column name.
I'll update the PR to check if the column matches the existing select.

Reply:
Added the extra check! 
I don't know if you like the early returns with the continue's. But it seems like that would be the most readable?
Let me know if you want it changed 😃